platform/src/apis/EducationBenchmarking.Platform.Api.Establishment/Db/Edubase.cs

```diff
@@ -0,0 +1,132 @@
+    public string TelephoneNum { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.NO_PUPIL)]
+    public float? NumberOfPupils { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.STAT_LOW)]
+    public int? StatutoryLowAge { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.STAT_HIGH)]
+    public int? StatutoryHighAge { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.HEAD_FIRST_NAME)]
+    public string HeadFirstName { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.NURSERY_PROVISION)]
+    public string NurseryProvision { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.HEAD_LAST_NAME)]
+    public string HeadLastName { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.OFFICIAL_6_FORM)]
+    public string OfficialSixthForm { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.SCHOOL_WEB_SITE)]
+    public string SchoolWebsite { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.OFSTED_RATING)]
+    public string OfstedRating { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.OFSTE_LAST_INSP)]
+    public string OfstedLastInsp { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.FINANCE_TYPE)]
+    public string FinanceType { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.OPEN_DATE)]
+    public string OpenDate { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.CLOSE_DATE)]
+    public string CloseDate { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.RELIGIOUS_CHARACTER)]
+    public string ReligiousCharacter { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.MAT_SAT)]
+    public string MatSat { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.IS_FEDERATION)]
+    public bool IsFederation { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.IS_PART_OF_FEDERATION)]
+    public bool IsPartOfFederation { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.FEDERATION_MEMBERS)]
+    public long[] FederationMembers { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.FEDERATION_UID)]
+    public long? FederationUid { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.FEDERATION_NAME)]
+    public string FederationName { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.FEDERATIONS_CODE)]
+    public long? FederationsCode { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.FEDERATION)]
+    public string Federation { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.GOV_OFFICE_REGION)]
+    public string GovernmentOfficeRegion { get; set; }
+
+    [JsonProperty(PropertyName = FieldNames.ESTAB_STATUS)]
+    public string EstablishmentStatus { get; set; }
+    
+    public class LocationDataObject
+    {
+        [JsonProperty(PropertyName = FieldNames.LOCATION_TYPE)]
+        public string type { get; set; }
+
+        [JsonProperty(PropertyName = FieldNames.LOCATION_COORDINATES)]
+        public string[] coordinates { get; set; }
+    }
+}
```

Comment:
Is the naming convention used for the _FieldNames_ attributes standard or personal preference? If the latter then I think we should have an agreed attribute naming convention used within the team to keep everything consistent.

Reply:
Naming convention should follow the language specific code styles. However actually naming on variables, methods, etc should be meaning without being overly verbose.